src/Oro/Bundle/CustomerBundle/Layout/DataProvider/FrontendCustomerUserFormProvider.php

```diff
@@ -51,7 +51,7 @@ public function getCustomerUserForm(CustomerUser $customerUser)
      */
     public function getForgotPasswordFormView(array $options = [])
     {
-        $options['action'] = $this->generateUrl(self::ACCOUNT_USER_RESET_REQUEST_ROUTE_NAME);
+        $options['action'] = isset($options['action']) ?: $this->generateUrl(static::ACCOUNT_USER_RESET_REQUEST_ROUTE_NAME);
```

Comment:
When I pass `$options['action'] = 'my_new_action'` to the method argument, the $options['action'] will be set to `true`.
What is the point of this changes?

Reply:
if in options['action'] you have an url (added in the controller), you haven't to create a custom file and extend the provider for only specifying a custom route name

Reply:
https://3v4l.org/cra11

Seems it should be something like
`$options['action'] ?? $this->generateUrl(static::ACCOUNT_USER_RESET_REQUEST_ROUTE_NAME)`